frontend/src/app/modules/grids/widgets/news/news.component.html

```diff
@@ -2,31 +2,26 @@
 
   <widget-menu [resource]="resource">
   </widget-menu>
-  
+
 </widget-header>
 <div class="grid--widget-content">
   <no-results *ngIf="noEntries" [title]="text.noResults">
   </no-results>
+
   <ul class="widget-box--arrow-links">
     <li class="-widget-box--arrow-multiline" *ngFor="let news of entries">
       <div>
-        <user-avatar *ngIf="news.author" [user]="news.author" data-class-list="avatar avatar-medium -spaced">
+        <user-avatar *ngIf="news.author" [user]="news.author" data-class-list="avatar" class="author-avatar">
```

Comment:
On the homescreen, the class `author-avatar` is directly on the `img`-tag, here it is on the `user-avatar`. Please decide for one option and use this one on both.